packages/web/src/js/utils/ComponentFunctions.ts

```diff
@@ -30,6 +30,15 @@ const enableDataTrigger = (
   });
 };
 
+const instanceAutoloader = (dataAttributeKey: string, component: typeof BaseComponent) => {
```

Comment:
I think this is a duplication of the code above. Why have you added this functionality?